symengine/tests/printing/test_printing.cpp

```diff
@@ -42,6 +42,12 @@ using SymEngine::complex_double;
 using SymEngine::BaseVisitor;
 using SymEngine::StrPrinter;
 using SymEngine::Sin;
+using SymEngine::integer_class;
+
+inline integer_class operator "" _z(const char* str)
+{
+    return integer_class(str);
+}
```

Comment:
Yes, I moved it because it was a feature introduced in gcc-4.7 and at that time we were still using gcc-4.6 and I wanted check in Travis that it still compiles. I'll move this back to SymEngine code itself.